.eslintrc.json

```diff
@@ -1,8 +1,39 @@
 {
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
```

Comment:
Each of these plugins, rules and parser should go into their relevant eslint config, not the default one.
There's more configs in each of the directories in the root of the project.

Reply:
Under what folder should the eslint typescript-related rules go?
(the ones that need to be translated tslint to eslint-equivalent) @JakeSidSmith

Reply:
Created `typescript` folder for this.